src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -19,70 +16,159 @@
             [status-im.chat.models.message :as models.message]
             [status-im.chat.commands.sending :as commands-sending]
             [status-im.constants :as constants]
-            [status-im.transport.utils :as transport.utils]
-            [taoensso.timbre :as log]
             [status-im.ui.screens.navigation :as navigation]
             [status-im.wallet.transactions :as wallet.transactions]))
 
 ;;;; FX
 
-(re-frame/reg-fx
- ::accept-transaction
- (fn [{:keys [masked-password id on-completed]}]
-  ;; unmasking the password as late as possible to avoid being exposed from app-db
-   (status/approve-sign-request id
-                                (security/unmask masked-password)
-                                on-completed)))
+(defn- send-ethers [params on-completed masked-password]
+  (status/send-transaction (types/clj->json params)
+                           (security/unmask masked-password)
+                           on-completed))
 
-(re-frame/reg-fx
- ::accept-transaction-with-changed-gas
- (fn [{:keys [masked-password id on-completed gas gas-price default-gas-price]}]
-   ;; unmasking the password as late as possible to avoid being exposed from app-db
-   (if gas
-     (status/approve-sign-request-with-args id
-                                            (security/unmask masked-password)
-                                            (money/to-fixed gas)
-                                            (money/to-fixed (or gas-price default-gas-price))
-                                            on-completed)
-     (status/approve-sign-request id
-                                  (security/unmask masked-password)
-                                  on-completed))))
-
-(defn- send-ethers [{:keys [web3 from to value gas gas-price]}]
-  (.sendTransaction (.-eth web3)
-                    (clj->js {:from from :to to :value value :gas gas :gasPrice gas-price})
-                    #()))
-
-(defn- send-tokens [{:keys [web3 from to value gas gas-price symbol chain]}]
+(defn- send-tokens [symbol chain {:keys [from to value gas gas-price]} on-completed masked-password]
   (let [contract (:address (tokens/symbol->token (keyword chain) symbol))]
-    (erc20/transfer web3 contract from to value {:gas gas :gasPrice gas-price} #())))
+    (status/send-transaction (types/clj->json
```

Comment:
Why not fn call anymore?

Reply:
sorry @jeluard what do you mean?

Reply:
Why the change from `transfer` to raw `send-transaction` ?

Reply:
we didn't change `transfer` we changed web3.js sendtransaction call on 'native' sendtransaction call, i mean we don't call `erc20/transfer` anymore, because we don't use web3 anymore, but functionality stay the same, we still send transaction with calling transfer function in contract

Reply:
whether it's web3 or json-rpc that function should be encapsulated. I see no reason why `erc20` ns couldn't have json-rpc calls, it already does for some token transfer history things